DataFormats/TotemReco/interface/TotemT2Digi.h

```diff
@@ -12,18 +12,24 @@
 class TotemT2Digi {
 public:
   TotemT2Digi() = default;
-  TotemT2Digi(unsigned char geo, unsigned char id, unsigned char marker, unsigned short le, unsigned short te);
+  TotemT2Digi(unsigned short id, unsigned char marker, unsigned short le, unsigned short te);
 
   void setLeadingEdge(unsigned short le) { lead_edge_ = le; }
   unsigned short leadingEdge() const { return lead_edge_; }
   void setTrailingEdge(unsigned short te) { trail_edge_ = te; }
   unsigned short trailingEdge() const { return trail_edge_; }
+  bool hasLE() const { return marker_ & 0x1; }
+  bool hasTE() const { return marker_ & 0x2; }
+  bool hasManyLE() const { return marker_ & 0x4; }
+  bool hasManyTE() const { return marker_ & 0x8; }
 
 private:
   /// Geo ID
-  unsigned char geo_id_{0};
+  //  unsigned char geo_id_{0};
```

Comment:
If this and `channel_id_` are not used, please remove them, dont just comment them out